examples/java/build.gradle

```diff
@@ -64,6 +64,7 @@ dependencies {
   implementation project(":sdks:java:extensions:avro")
   implementation project(":sdks:java:extensions:google-cloud-platform-core")
   implementation project(":sdks:java:extensions:python")
+  implementation project(":sdks:java:extensions:sql")
```

Comment:
This would introduce  the heavy beam-sql dependencies to examples-java. Consider create a subproject under example

Reply:
discussed offline, the goal is to add a Beam SQL example to plaground, where SQL dependencies already present (added by go/beampr/26227).

Rearranging example jars is nontrivial to playground, so good for now